app/controllers/microposts_controller.rb

```diff
@@ -0,0 +1,40 @@
+class MicropostsController < ApplicationController
+  before_action :logged_in_user, only: %i(create destroy)
+  before_action :find_micropost, only: :destroy
+
+  def create
+    @micropost = current_user.microposts.build(micropost_params)
```

Comment:
so sánh method `build` vs `new`
```suggestion
    @micropost = current_user.microposts.build micropost_params
```

Reply:
New method
New method truyền data vào nhưng không save. Muốn tạo object thì cần kết hợp với cả save method nữa.

Build method
Build method giống New ở chỗ đều có khả năng tạo object mới.
Nhưng khi tạo biến mới thì Build luôn đi cùng với user_id, điều mà New không làm được.